src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -552,6 +555,16 @@ int get_model_prefer_threads(const int num_streams,
     const int sockets = get_num_sockets();
     auto model_prefer = 0;
     if (-1 == config.modelPreferThreads) {
+#if (defined(OPENVINO_ARCH_ARM64) && defined(__linux__))
+        config.modelPreferThreads = 8;
+        if (dnnl::impl::cpu::aarch64::mayiuse(dnnl::impl::cpu::aarch64::cpu_isa_t::isa_all) ||
+            dnnl::impl::cpu::aarch64::mayiuse(dnnl::impl::cpu::aarch64::cpu_isa_t::sve_512) ||
+            dnnl::impl::cpu::aarch64::mayiuse(dnnl::impl::cpu::aarch64::cpu_isa_t::sve_384) ||
+            dnnl::impl::cpu::aarch64::mayiuse(dnnl::impl::cpu::aarch64::cpu_isa_t::sve_256) ||
+            dnnl::impl::cpu::aarch64::mayiuse(dnnl::impl::cpu::aarch64::cpu_isa_t::sve_128)) {
```

Comment:
What is the idea behind that condition? Find ARM CPUs with SVE isa?

Reply:
The idea is to decide threads per stream by ARM ISA type identified by OneDNN.

The current analysis results show that for asimd ARM, 8 threads are better, while for sve_128 ARM, 16 threads are better.

Reply:
1. Based on mayiuse impl it is enough to check dnnl::impl::cpu::aarch64::mayiuse(dnnl::impl::cpu::aarch64::cpu_isa_t::sve_128) only to understand if SVE isa is supported or not.
![image](https://github.com/user-attachments/assets/de26a5e5-0856-4208-b101-07ad0c6dd7b8)
2. Shouldn't we update `isaSpecificThreshold` based on ISA instead of `modelPreferThreads` to preserve overall math model?

Reply:
Updated. @dmitry-gorokhov please review again.